packages/text-annotator/src/highlight/index.ts

```diff
@@ -1,2 +1,4 @@
+export * from './HighlightPainter';
+export * from './HighlightStyle';
```

Comment:
I think that those files also should be re-exported because they are part of a public API and will be utilized by users who want to create custom painters or extend the styles with their app-specific props